File: scala/scala-impl/src/org/jetbrains/sbt/project/SbtProjectSystem.scala
package org.jetbrains.sbt
package project

import com.intellij.openapi.externalSystem.model.ProjectSystemId
import org.jetbrains.plugins.scala.NlsString

/**
 * @author Pavel Fatin
 */
object SbtProjectSystem {
  val Id = new ProjectSystemId("SBT", NlsString.force(Sbt.Name))
}

File: sbt/sbt-impl/src/org/jetbrains/sbt/project/SbtProjectSystem.java
package org.jetbrains.sbt.project;

import com.intellij.openapi.externalSystem.model.ProjectSystemId;
import org.jetbrains.plugins.scala.NlsString;
import org.jetbrains.sbt.Sbt;

public final class SbtProjectSystem {
    public static final ProjectSystemId Id = new ProjectSystemId("SBT", NlsString.force(Sbt.Name()));
}
